### PR TITLE
Popover menu: update tracking attributes + move under Explore/logs

### PR DESCRIPTION
**Pull request by matyax**
- Moved to Explore/logs.
- Updated tracking attributes.

**Which issue(s) does this PR fix?**:

Part of https://github.com/grafana/grafana/issues/76129

**Special notes for your reviewer:**

Nothing in particular.

### CODE REVIEW

File: public/app/features/explore/Logs/PopoverMenu.tsx [hunk 1]
```diff
@@ -87,8 +87,8 @@ export const PopoverMenu = ({
 function track(action: string, selectionLength: number, dataSourceType: string | undefined) {
   reportInteraction(`grafana_explore_logs_popover_menu`, {
     action,
-    selection_length: selectionLength,
-    ds_type: dataSourceType || 'unknown',
+    selectionLength: selectionLength,
+    datasourceType: dataSourceType || 'unknown',
```
Comment (svennergr) [inline public/app/features/explore/Logs/PopoverMenu.tsx:91]: Thank you @ivanahuckova 🙇

**Review by svennergr — APPROVED** [review]
Just a thought, do we want to add a safeguard in `popoverMenuSupported` to only support it when `app === CoreApp.Explore` for now?

**Comment by matyax** [comment]
> Just a thought, do we want to add a safeguard in popoverMenuSupported to only support it when app === CoreApp.Explore for now?

What are other possible values for `app` in that context?

**Comment by svennergr** [comment]
> > Just a thought, do we want to add a safeguard in popoverMenuSupported to only support it when app === CoreApp.Explore for now?
> 
> What are other possible values for `app` in that context?

https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/app.ts#L17-L26

**Comment by matyax** [comment]
Oh, the check is in `LogRows.tsx` not in `Explore.tsx` or `Explore/Logs.tsx`. I think it makes total sense.

**Comment by matyax** [comment]
Great idea @svennergr , done.

**Pull request merged**